src/lib/Ledger/index.spec.ts

```diff
@@ -1,12 +1,12 @@
 import { LedgerKeyManager } from '.'
-import { KeyManager } from '../KeyManager'
+import { KeyManager } from '../../KeyManager/KeyManager'
```

Comment:
As a general rule I think `index.ts` for `KeyManager` within a directory named `KeyManager` is a good style, or we could decide to go with exporting the module as a default from the index. Either way, let's make it consistent

Reply:
Ahhh that was just the editor

Reply:
> As a general rule I think index.ts for KeyManager within a directory named KeyManager is a good style
100%